### PR TITLE
Revert "Make fiddling with search from the console easier"

### PR DESCRIPTION
**Pull request by jscn**
Reverts ask-izzy/ask-izzy#544

The PMs want to do several releases without this change, so it's easiest to just undo the merge for now.

### CODE REVIEW

File: src/client.js [hunk 1]
```diff
@@ -8,10 +8,8 @@ import storage from "./storage";
 import routes from "./routes";
 import sendEvent from "./google-tag-manager";
 import searchTest from "./search-test";
-import categories from "./constants/categories";
 
 window.searchTest = searchTest;
-window.categories = categories;
```
Comment (mucow) [inline src/client.js:14]: Was this all there was to revert? It just enabled someone to be able to fiddle with some of the data on the client?

Comment (jscn) [inline src/client.js:14]: Yes, that's the whole changeset.

Comment (jscn) [inline src/client.js:14]: Just doing as asked :p

Comment (mucow) [inline src/client.js:14]: Lol ++

**Pull request merged**